runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -822,6 +846,59 @@ func s:HandleDisasmMsg(msg)
   endif
 endfunc
 
+func s:ParseVarinfo(varinfo)
+  let dict = {}
+  let nameIdx = matchstrpos(a:varinfo, '{name="\([^"]*\)"')
+  let dict['name'] = a:varinfo[nameIdx[1] + 7 : nameIdx[2] - 2]
+  let typeIdx = matchstrpos(a:varinfo, ',type="\([^"]*\)"')
+  let dict['type'] = a:varinfo[typeIdx[1] + 7 : typeIdx[2] - 2]
+  let valueIdx = matchstrpos(a:varinfo, ',value="\(.*\)"}')
+  if valueIdx[1] == -1
+    let dict['value'] = 'Complex value'
+  else
+    let dict['value'] = a:varinfo[valueIdx[1] + 8 : valueIdx[2] - 3]
+  endif
+  return dict
+endfunc
+
+func s:SpacesString(num)
+  let i = 0
+  let ret = ''
+  while i < a:num
+    let ret = ret . ' '
+    let i += 1
+  endwhile
+  return ret
+endfunc
+
+func s:HandleVariablesMsg(msg)
+  let curwinid = win_getid(winnr())
+  if win_gotoid(s:varwin)
+
+    silent normal! gg0"_dG
+    let spaceBuffer = 20
+    call setline(1, 'Type' .
+	  \ s:SpacesString(16) .
+	  \ 'Name' .
+	  \ s:SpacesString(16) .
+	  \ 'Value')
```

Comment:
Use double dots (..) instead of a single dot (.) for string concatenation.  It will be more readable, if you use string interpolation:  `$'Type{repeat(' ', 16)}Name{repeat(' ', 16)}Value`

Reply:
Single dots are used throughout most of the file for string concatenation. Should I change it in all the locations?

Reply:
it is a code style and vim9 may require double dots, vs for legacy vim script, as long as clear enough should be ok.
// IMO: unless refactor/rewrite to vim9 one day, perhaps better if kept its current style.
// or if you have time then change its all, but actually indent was not all same style there.

Reply:
We can address the single dot to double dots conversion in a separate PR.  You can leave that out of this PR.